automation_tools/__init__.py

```diff
@@ -850,6 +850,80 @@ def iso_install(
         return installer_options
 
 
+def iso_install_5(
```

Comment:
Please name the task `satellite5_iso_insall`, now that we will be installing different product versions will be better to prefix the tasks with the product name + major version.